modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -89,6 +90,41 @@ object method {
     }
   }
 
+  case class RunRollback(
+    keyStore: StorePath,
+    alias: KeyAlias,
+    password: Password,
+    dbConfig: DBConfig,
+    httpConfig: HttpConfig,
+    environment: AppEnvironment,
+    snapshotConfig: SnapshotConfig,
+    whitelistingPath: Option[Path],
+    collateralAmount: Option[Amount],
+    rollbackHash: Hash
+  ) extends Run
+
+  object RunRollback extends WithOpts[RunRollback] {
+
+    val whitelistingPathOpts: Opts[Option[Path]] = Opts.option[Path]("whitelisting", "").orNone
+
+    val rollbackHashOpts: Opts[Hash] = Opts.argument[Hash]("rollbackHash")
```

Comment:
I'd also allow for rollback from ordinal or the latest ordinal if nothing is specified. What do you think?

Reply:
Yeah we could add it as an additional feature, but it requires a bit more work and possibly some protection, so I didn't try to include it here. I can't create a ticket for it though.